src/Cookiebar.php

```diff
@@ -215,7 +213,7 @@ public static function parseCookiebarTemplate(CookiebarModel $objConfig): string
 
         $objTemplate = new FrontendTemplate($objConfig->template);
 
-        $cssID = unserialize($objConfig->cssID);
+        $cssID = StringUtil::deserialize($objConfig->cssID);
```

Comment:
```suggestion
        $cssID = StringUtil::deserialize($objConfig->cssID, true) + [null, null];
```

This way $cssID will always be an array with (at least) two elements.